test/Test/Database/LSMTree/StateMachine.hs

```diff
@@ -1687,6 +1687,16 @@ shrinkActionWithVars _ctx _st = \case
       | let f k = (k, R.Delete)
       ]
 
+    Mupserts kvs tableVar -> [
+        Some $ Mupserts kvs' tableVar
+      | kvs' <- QC.shrink kvs
+      ]
```

Comment:
We could also try shrinking to inserts, which we already do for `Updates` and in the prototype, but it's a little more complicated here, since the whole batch has to become a batch of inserts. Maybe still useful to try?

Reply:
Ok, I've added shrinking towards Updates using R.Mupsert